policyengine_us/parameters/gov/states/oh/tax/income/credits/lump_sum_distribution/expected_remaining_years.yaml

```diff
@@ -397,89 +395,132 @@ brackets:
       2022-01-01: 1.4
   - threshold:
       2021-01-01: 94
+      2022-01-01: .inf
     amount:
       2021-01-01: 3.9
+      2022-01-01: 1.4
   - threshold:
       2021-01-01: 95
+      2022-01-01: .inf
     amount:
       2021-01-01: 3.7
+      2022-01-01: 1.4
   - threshold:
       2021-01-01: 96
+      2022-01-01: .inf
     amount:
       2021-01-01: 3.4
+      2022-01-01: 1.4
   - threshold:
       2021-01-01: 97
+      2022-01-01: .inf
     amount:
       2021-01-01: 3.2
+      2022-01-01: 1.4
   - threshold:
       2021-01-01: 98
+      2022-01-01: .inf
     amount:
       2021-01-01: 3.0
+      2022-01-01: 1.4
   - threshold:
       2021-01-01: 99
+      2022-01-01: .inf
     amount:
       2021-01-01: 2.8
+      2022-01-01: 1.4
   - threshold:
       2021-01-01: 100
+      2022-01-01: .inf
     amount:
       2021-01-01: 2.7
+      2022-01-01: 1.4
   - threshold:
       2021-01-01: 101
+      2022-01-01: .inf
     amount:
       2021-01-01: 2.5
+      2022-01-01: 1.4
   - threshold:
       2021-01-01: 102
+      2022-01-01: .inf
     amount:
       2021-01-01: 2.3
+      2022-01-01: 1.4
   - threshold:
       2021-01-01: 103
+      2022-01-01: .inf
     amount:
       2021-01-01: 2.1
+      2022-01-01: 1.4
   - threshold:
       2021-01-01: 104
+      2022-01-01: .inf
     amount:
       2021-01-01: 1.9
+      2022-01-01: 1.4
   - threshold:
       2021-01-01: 105
+      2022-01-01: .inf
     amount:
       2021-01-01: 1.8
+      2022-01-01: 1.4
   - threshold:
       2021-01-01: 106
+      2022-01-01: .inf
     amount:
       2021-01-01: 1.6
+      2022-01-01: 1.4
   - threshold:
       2021-01-01: 107
+      2022-01-01: .inf
     amount:
       2021-01-01: 1.4
   - threshold:
       2021-01-01: 108
+      2022-01-01: .inf
     amount:
       2021-01-01: 1.3
+      2022-01-01: 1.4
   - threshold:
       2021-01-01: 109
+      2022-01-01: .inf
     amount:
       2021-01-01: 1.1
+      2022-01-01: 1.4
   - threshold:
       2021-01-01: 110
+      2022-01-01: .inf
     amount:
       2021-01-01: 1.0
+      2022-01-01: 1.4
   - threshold:
       2021-01-01: 111
+      2022-01-01: .inf
     amount:
       2021-01-01: 0.9
+      2022-01-01: 1.4
   - threshold:
       2021-01-01: 112
+      2022-01-01: .inf
     amount:
       2021-01-01: 0.8
+      2022-01-01: 1.4
   - threshold:
       2021-01-01: 113
+      2022-01-01: .inf
```

Comment:
could you test this in 2022 / 2023 just to make sure we get expected values for higher thresholds